docs/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "vitepress dev src",
+    "build": "vitepress build src"
+  },
+  "devDependencies": {
+    "vitepress": "^0.7.4"
```

Comment:
Shouldn't we use the code in the repo itself to deploy the docs? I think that also serves as a form of e2e test.

Reply:
Oh yes I think we should! Just wasn't sure how...? I was looking into VuePress code and seems like it was doing this way... so...

Do you have any idea how we can do this 🤔 ?

Reply:
@yyx990803 OK I think I fixed it. Not sure if this is the cleanest way to go. So now we have `docs/package.json` that only contains scripts. This is because if we reference `bin/vitepress.js` from root `package.json`, it tries to optimize node specific modules and crashes.

While we don't have a way to pass `optimizeDev` option to underlining Vite, I went this way.